src/components/molecules/CategoryUpdate/index.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <div class="category-update">
+    <app-heading :level="1">カテゴリー管理</app-heading>
+    <app-router-link
+      class="category-management-post__detail"
```

Comment:
[nits]

class 名ですが、`post` となっているところは、`update` としたほうが規則性があるかと思います！

```suggestion
      class="category-management-update__detail"
```

他の箇所も同様なので修正をお願いします！